pkg/rke/k8s_rke_system_images.go

```diff
@@ -9965,10 +9965,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			AciOpenvSwitchContainer:   "noiro/openvswitch:6.0.3.2.81c2369",
 			AciControllerContainer:    "noiro/aci-containers-controller:6.0.3.2.81c2369",
 			PodInfraContainer:         "rancher/mirrored-pause:3.7",
-			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.9.5-rancher1",
+			Ingress:                   "rancher/nginx-ingress-controller:nginx-1.9.6-rancher1",
 			IngressBackend:            "rancher/mirrored-nginx-ingress-controller-defaultbackend:1.5-rancher1",
 			IngressWebhook:            "rancher/mirrored-ingress-nginx-kube-webhook-certgen:v20231011-8b53cabe0",
```

Comment:
We need to update webhook certgen as well, 1.9.6 uses latest release `v20231226-1a7112e06` https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.9.6

Reply:
Created image-mirror [PR](https://github.com/rancher/image-mirror/pull/581) to add the mirrored image.